plugins/lighthouse/.eslintrc.js

```diff
@@ -1,3 +1,10 @@
 module.exports = {
+  plugins: [
+  "react-hooks"
```

Comment:
may want to set your editor to run prettier on save

Reply:
This was actually me. I added the eslint-react-hooks plugin. I don't know if it is ok to leave it in

Reply:
Right, that's OK, I just meant that it wasn't at the right indentation level so your editor may need some reconfiguring (or run reformat manually) :)

Reply:
Freben is talking about the indentation on line 3, it's not far enough.

Regarding the rule I actually thought this was a part of `spotify/web-scripts`, @fastfrwrd do you want to add react-hooks there? I find it catches a lot of errors.

Reply:
huh, I really thought it was already in there! I've opened https://github.com/spotify/web-scripts/issues/305 to track.